requirements.txt

```diff
@@ -0,0 +1,212 @@
+absl-py==0.14.1
+alabaster==0.7.8
+alembic==1.4.1
+antlr4-python3-runtime==4.8
+apturl==0.5.2
+attrs==21.2.0
+Babel==2.6.0
+backports.entry-points-selectable==1.1.0
+beautifulsoup4==4.10.0
+black @ git+git://github.com/psf/black@2f3fa1f6d0cbc2a3f31c7440c422da173b068e7b
```

Comment:
Why to we refer to this [commit](https://github.com/psf/black/commit/2f3fa1f6d0cbc2a3f31c7440c422da173b068e7b)?
Shouldn't we refer to some version? like `21.10b0` or `stable`...

Reply:
Please check for an updated push. I've recreated the requirements.txt file strictly from the docker files instructions.